code/modules/awaymissions/corpse.dm

```diff
@@ -21,6 +21,10 @@
 	density = 1
 	anchored = 1
 
+/obj/effect/mob_spawn/New()
```

Comment:
Never assume anything with my 2012 era shitcode (or my modern shitcode for that matter)